plugins/explore/src/components/GroupsExplorerContent/GroupsDiagram.tsx

```diff
@@ -46,6 +46,12 @@ const useStyles = makeStyles((theme: BackstageTheme) => ({
     flex: 1,
     minHeight: 0,
   },
+  graphWrapper: {
+    position: 'relative',
+    flex: 1,
+    minHeight: 0,
+    display: 'flex',
+  },
```

Comment:
Do you see a way to get this to fill up the rest of the empty space on the page without using any magic numbers? I feel there's enough vertical space that the best solution to this problem would be if the graph simply fills out the rest of the page and there's no vertical scrolling at all. What do you think?

Reply:
I think the current solution works pretty well actually, but if you think that would be a better solution I will look into it. 👍 

If we are going with that solution I guess we will be going back to the previous scroll behaviour, otherwise it will be difficult for the user to scroll down. I will also remove the added background to the graph, so it doesn't blend together with the sidebar as it fills the empty space.

Reply:
I haven't checked, but does this solve the original issue of not being able to scroll back up if the mouse is hovered above the diagram?

What I'm proposing is the make the graph fill the page in a way where it is not possible to scroll at all

Reply:
Yeah, I see. When reading what people were saying it seemed that the underlying issue was that it's unexpected behaviour when you can scroll down when hovering the graph, but not scroll up. What the current solution aims to do is to make it clear that you should not expect to scroll up or down when hovering the graph (like google maps for instance),  and it does this by adding a background to the graph and removing scroll behaviour when hovering the graph.